Makefile

```diff
@@ -18,8 +18,17 @@ BUILD_DIR=release
 BUILD_TYPE=Release
 TREAT_WARNINGS_AS_ERRORS ?= 1
 ENABLE_WALL ?= 1
-WARNINGS_AS_ERRORS=-DTREAT_WARNINGS_AS_ERRORS=${TREAT_WARNINGS_AS_ERRORS}
-ENABLE_ALL_WARNINGS=-DENABLE_ALL_WARNINGS=${ENABLE_WALL}
+
+# Control whether to build unit tests. By default set to "ON"; set to
+# "OFF" to disable.
+VELOX_BUILD_TESTING?= "ON"
```

Comment:
nit: space after variable name, for consistency?

Reply:
> nit: space after variable name, for consistency?

fixed